src/classes/ERR_ExceptionHandler.cls

```diff
@@ -112,4 +113,5 @@ public with sharing class ERR_ExceptionHandler {
 
     public class AfflAccountNoRecordTypeException extends Exception {}
     public class AffAccountRecordTypeMappingsException extends Exception {}
+    public class nestedPlanRequirementPPException extends Exception {}
```

Comment:
@xinnanlisf @lifewithryan  This might be me being super nit-picky, and I know we want to move this PR forward, but shouldn't class names begin with a capital letter?

Reply:
Yes, it should be. I'll do a run through and change it where it needs to be changed since Xinnan is on PTO.

Reply:
done